render_files.py

```diff
@@ -50,7 +51,10 @@ def update_bmh_files(ignition_file, cluster_id, inventory_endpoint):
             data = json.load(file_obj)
             storage_files = data['storage']['files']
             # since we don't remove file for now, we don't need to iterate through copy
+            # for file_data in storage_files[:]:
```

Comment:
comments are for future use?

Reply:
yes, this is preparations for 4.6 in 2 weeks